go-controller/pkg/metrics/ovnkube_controller.go

```diff
@@ -262,17 +263,37 @@ var metricEgressFirewallCount = prometheus.NewGauge(prometheus.GaugeOpts{
 var metricANPCount = prometheus.NewGauge(prometheus.GaugeOpts{
 	Namespace: MetricOvnkubeNamespace,
 	Subsystem: MetricOvnkubeSubsystemController,
-	Name:      "admin_network_policy_custom_resource_total",
+	Name:      "admin_network_policies",
 	Help:      "The total number of admin network policies in the cluster",
 })
 
 var metricBANPCount = prometheus.NewGauge(prometheus.GaugeOpts{
 	Namespace: MetricOvnkubeNamespace,
 	Subsystem: MetricOvnkubeSubsystemController,
-	Name:      "baseline_admin_network_policy_custom_resource_total",
-	Help:      "The total number of baseline admin network policies (0 or 1) in the cluster",
+	Name:      "baseline_admin_network_policies",
+	Help:      "The total number of baseline admin network policies in the cluster",
 })
 
+var metricANPDBObjects = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+	Namespace: MetricOvnkubeNamespace,
+	Subsystem: MetricOvnkubeSubsystemController,
+	Name:      "admin_network_policies_db_objects",
+	Help:      "The total number of OVN NBDB objects (table_name) owned by AdminNetworkPolicy controller in the cluster"},
+	[]string{
+		"table_name",
```

Comment:
can `table_name` values appear/disappear over the lifetime of the controller? If yes, you need to remove stale labels.

Reply:
table_names disappearing is going to be a very very rare thing, but appearing is totally possible although I don't anticipate more than 1 or 2 max appaaring to this list.
So currently we are trying to go with `ACLs` and `AddressSets` as the labels, and we will add `PortGroups` as well if need be.

Reply:
>  If yes, you need to remove stale labels.

is there something special I need to do for this now or did you mean more for the future *if* we remove a name we must remember to clean it up ? (sorry for the noob question)

Reply:
> if we remove a name we must remember to clean it up ?

Exactly this. If at some point, you have the "AddressSets" table reporting a value of 100 and later the table goes away (without a restart of the controller), you don't want your metrics endpoint to continue reporting the value. In practice you'd need to call `m.DeleteLabelValues("AddressSets")`. 
But looking at the code again, the label values are hardcoded so you should be fine.